tests/test_queuing.py

```diff
@@ -341,8 +341,8 @@ def test_circuit_integration(self):
             + "==========\n"
             + "PauliY(wires=[0])\n"
             + "PauliY(wires=[1])\n"
-            + "RZ(tensor(0.4, requires_grad=True), wires=[0])\n"
-            + "RZ(tensor(0.4, requires_grad=True), wires=[1])\n"
+            + "RZ(0.4, wires=[0])\n"
+            + "RZ(0.4, wires=[1])\n"
```

Comment:
oh this is a nice improvement! Although I'm curious what is causing this to change?

Reply:
Well, since there is no longer an implicit conversion of QNode arguments to the Tensor class, the gate arguments in this test remain scalars instead of printing as tensors due to the conversion.

Reply:
Nice!